docs/feature-gates.md

```diff
@@ -35,6 +35,7 @@ edit the Agent configuration in the
 | `AntreaProxy`                 | Agent              | `true`  | GA    | v0.8          | v0.11        | v1.14      | Yes                | Must be enabled for Windows.                  |
 | `EndpointSlice`               | Agent              | `true`  | GA    | v0.13.0       | v1.11        | v1.14      | Yes                |                                               |
 | `TopologyAwareHints`          | Agent              | `true`  | Beta  | v1.8          | v1.12        | N/A        | Yes                |                                               |
+| `ServiceTrafficDistribution`  | Agent              | `true`  | Alpha | v2.2          | N/A          | N/A        | Yes                |                                               |
```

Comment:
The default is `true`? If the change is straightfoward and we are confident in its quality and want it to be enabled by default, the stage should be Beta to be consistent.

Reply:
+1 I personally support going straight to Beta and enabling it by default.
K8s 1.31 was just released and the feature has been promoted already

Reply:
My bad, this should be `false`. I copied this from the line above it and forgot to modify the default value.

Reply:
@hongliangl It may still make sense to introduce this as Beta right away:
- there is no requirement that feature gates always go through Alpha stage
- code changes are small and mostly ported over from kube-proxy
- this change should have no impact unless users set `.spec.trafficDistribution`
- the feature gate is already Beta / enabled by default in the latest K8s version, as it only stayed in Alpha stage for one minor release

Reply:
It was done as you said.